Observer/Admin/DashboardInstantiation.php

```diff
@@ -139,7 +140,7 @@ public function execute(Observer $observer) : void
                 'moduleVersion' => $moduleVersion,
                 'platformVersion' => (string) $this->productMetadata->getVersion()
             ];
-
+            // die(json_encode($installRequestData));
```

Comment:
Leftover from debugging?

Reply:
did you ever consider that I just wanted magento to die?

Reply:
<img width="576" alt="Screen Shot 2020-07-31 at 1 24 38 PM" src="https://user-images.githubusercontent.com/1248480/89074414-34bed000-d331-11ea-9fab-ce920cdf3d56.png">